packages/protocol/migrations/07_reserve_sender_multisig.ts

```diff
@@ -0,0 +1,18 @@
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
+import { deploymentForProxiedContract } from '@celo/protocol/lib/web3-utils'
+import { config } from '@celo/protocol/migrationsConfig'
+import { ReserveSpenderMultiSigInstance } from 'types'
+
+const initializeArgs = async (): Promise<any[]> => {
+  return [
+    config.reserveSpenderMultiSig.signatories,
+    config.reserveSpenderMultiSig.numRequiredConfirmations,
+  ]
+}
+
+module.exports = deploymentForProxiedContract<ReserveSpenderMultiSigInstance>(
+  web3,
+  artifacts,
+  CeloContractName.ReserveSpenderMultiSig,
+  initializeArgs
```

Comment:
One subtlety here that I neglected to mention is that ownership of the proxy should be turned over to itself the MultSig by setting the proxy owner to the proxy address. That will allow the MultiSig signatories to upgrade the MultiSig.

You should be able to do this in the `then` callback here, where the address of the proxy will be `reserveSpenderMultiSig.address`, where `reserveSpenderMultiSig` is the first argument.